lib/linguist/heuristics.yml

```diff
@@ -825,6 +825,10 @@ disambiguations:
     pattern: '&ANALYZE-SUSPEND _UIB-CODE-BLOCK _CUSTOM _DEFINITIONS'
   - language: CWeb
     pattern: '^@(<|\w+\.)'
+- extensions: ['.wdl']
+  rules:
+  - language: WDL
+    pattern: '^version'
```

Comment:
Why are you adding this? This change adds no value as this is the only language that uses this extension so the heuristic will never be used.

This PR is about changing the grammar so no other changes are needed.

Reply:
Yeah, I see now. I saw mention of updating the heuristics in the PR template, and, even though it was in the section for a new package, I figured it was required and was simply missed because the WDL language hadn't been updated in so long. That said, the wording does imply that it's only needed to help delineate between packages of the same extension.

I've removed this.